### PR TITLE
Added string comment for interactive user

### PR DESCRIPTION
**Pull request by 2020mt93231**
Please add this changes

### CODE REVIEW

File: division.py [hunk 1]
```diff
@@ -12,4 +12,5 @@ def division(num1, num2):
 
     num1 = int(input("Enter number"))
     num2 = int(input("Enter number"))
+    print("Division of two number is:")
```
Comment (amyvrm) [inline division.py:15]: Please write in one line

Comment (amyvrm) [reply division.py:15]: both output and result

**Review by amyvrm — CHANGES_REQUESTED** [review]

File: division.py (hunk 1 above)
Comment (2020mt93231) [reply division.py]: I made the change

Comment (amyvrm) [reply division.py]: looks good

**Pull request merged**